impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -54,6 +54,31 @@ Matrix MakeAnchorScale(const Point& anchor, Vector2 scale) {
          Matrix::MakeTranslation({-anchor.x, -anchor.y, 0});
 }
 
+void SetTileMode(SamplerDescriptor* descriptor,
+                 const ContentContext& renderer,
+                 Entity::TileMode tile_mode) {
+  switch (tile_mode) {
+    case Entity::TileMode::kDecal:
+      if (renderer.GetDeviceCapabilities().SupportsDecalSamplerAddressMode()) {
```

Comment:
Could you add a todo/issue for emulating this when it's not supported? The halo won't render correctly on some devices w/ the current technique without this.

We already have a specialized pipeline that the old blur uses for this w/ the same vertex/uniform layouts (`GaussianBlurDecalPipeline`).

Reply:
Done: https://github.com/flutter/flutter/issues/139773

How do we look up availability on gpuinfo.org?  I had a hard time finding it.  It's just available to everyone on Vulkan right?  The check is just for OpenGLES?

Reply:
Yeah CLAMP_TO_BORDER is supported for Metal and our min Vulkan profile without any extensions. For OpenGLES we check for `GL_EXT_texture_border_clamp` _or_ `GL_NV_texture_border_clamp`.

<img width="924" alt="image" src="https://github.com/flutter/engine/assets/919017/02f454b1-a623-4ec5-baf4-ac942464b369">

This looks worse than the situation actually is, though. I'm pretty sure "coverage" on gpuinfo.org is just "percentage of devices in the database" and doesn't weight for an estimation of actual usage. And of course, there are ancient devices on the list that Flutter doesn't support.

Reply:
Oh yea, good point.  Over time the usefulness of gpuinfo.org will wane if they don't have some sort of filter for old devices.  Something like filtering based on operating system would probably work since the OS developers will start dropping support.